chapter1/pipeline.md

```diff
@@ -142,3 +142,37 @@ Pipeline.play(pid)
 ```
 
 The given `.mp3` file should be played on default device in your system.
+
+## Connecting `push` pad to `pull` pad
+
+If you want to connect two pads, namely a pad that is in `push` mode to one that 
+is in `pull` mode, you MUST use a toilet. The toilet is a container that
+stores buffer send by `push` element and allows `pull` element to
+place demands. Creating a toilet is quite a simple feat. 
+When linking pads just use the third slot of the tuple.
```

Comment:
I'm not sure we can speak about slot here, it suggests that slot was always there while actually, we changed 2-tuple into 3-tuple. I would rephrase this sentence into just: `When linking pads, just add info about toilet to a proper input pad`.